documentation/docs/validator_recovery.md

```diff
@@ -0,0 +1,88 @@
+### Overview
+
+This document describes the steps necessary to ensure that a validator node can be restored in case its state is lots. In order to achieve this, it is necessary that the following data is safely backed up:
+
+* Tendermint validator key
+* Axelar validator mnemonic
+* Axelar proxy mnemonic
+* Tofnd mnemonic
+
+Besides the data described above, it will also be necessary to retrieve the *recovery data* associated to all the key shares that the validator was responsible for maintaining.
+
+### Tendermint validator key
+
+The Tendermint validator key is created when a node is launched for the first time.
+It can be found within the node's container at `/root/.axelar/config/priv_validator_key.json` (or from the node's root directory at `$ROOT_DIRECTORY/.core/config/priv_validator_key.json`).
+Its contents should look something like:
+
+```
+{
+  "address": "98AF6E5D52BBB62BE6717DE8C55F16F5C013D7BE",
+  "pub_key": {
+    "type": "tendermint/PubKeyEd25519",
+    "value": "CcspC1QDG8vz7kIW/7nPvqQ35XFJ5JLn5+li2WshP+o="
+  },
+  "priv_key": {
+    "type": "tendermint/PrivKeyEd25519",
+    "value": "VCG8TTeOSv+n9TOyy465CnUQALnoD/WJG9bloPGX0XUJyykLVAMby/PuQhb/uc++pDflcUnkkufn6WLZayE/6g=="
+  }
+}
+```
+
```

Comment:
missing: what to do with this

Reply:
first I need to adjust the scripts. That is why the info is not included yet. I should have marked it as a draft PR